### PR TITLE
pie-microsite@v1.48.0 - Colour global token descriptions

### PR DESCRIPTION
**Pull request by jamieomaguire**
v1.48.0
------------------------------
*January 27, 2023*

### Added
- Global token descriptions for colour page

### CODE REVIEW

**Review by FayeCarter — APPROVED** [review]

File: apps/pie-microsite/src/_11ty/shortcodes/tokensTable/tokenTypes/colour.js [hunk 1]
```diff
@@ -65,13 +65,23 @@ const buildColorExample = token => {
...
+ * Builds the overall token description element for global and alias Colour tokens.
+ * @param {*} tokenMetadata the metadata for the token. data such as descriptions
+ * @returns {string} - the description HTML string
+ */
+const buildColorDescription = tokenMetadata => {
     let description = '';
-    if (tokenMetadata.description) {
+
+    if (tokenMetadata.globalToken) {
+        // It's an alias colour token
```
Comment (raoufswe) [inline apps/pie-microsite/src/_11ty/shortcodes/tokensTable/tokenTypes/colour.js:77]: i think you meant the other way around?

Comment (jamieomaguire) [reply apps/pie-microsite/src/_11ty/shortcodes/tokensTable/tokenTypes/colour.js:78]: it's the correct way around but I've added a comment to give better context :)

**Review by raoufswe — APPROVED** [review]

**Review by FayeCarter — DISMISSED** [review]

File: apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js [hunk 2]
```diff
@@ -156,14 +156,15 @@ const buildTokenListElements = ({
...
-<ul class="c-tokensTable-list">
-  ${listElements.join('')}
-</ul>`;
+const buildTokensList = listElements => deindentHTML(`
+    <div class="c-tokensTable-row u-spacing-e--top u-showAboveWide c-tokensTable-heading">
+        <span>Example</span>
+        <span>Description</span>
+        <span>Token name</span>
+        </div>
+        <ul class="c-tokensTable-list">
```
Comment (fernandofranca) [inline apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js:165]: Indentation here seems a bit off.

Comment (jamieomaguire) [reply apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js:165]: fixed

**Review by raoufswe — APPROVED** [review]

**Review by fernandofranca — APPROVED** [review]

**Pull request merged**